src/main/interface/settings.c

```diff
@@ -729,6 +729,9 @@ const clivalue_t valueTable[] = {
     { "i_yaw",                      VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_YAW].I) },
     { "d_yaw",                      VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_YAW].D) },
 
+    { "vbat_throttle_compensation_enabled", VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_PID_PROFILE, offsetof(pidProfile_t, vbatThrottleCompensationEnabled) },
+    { "vbat_throttle_compensation_voltage", VAR_UINT8 | PROFILE_VALUE, .config.minmax = { 20, 50 }, PG_PID_PROFILE, offsetof(pidProfile_t, vbatThrottleCompensationVoltage) },
```

Comment:
The range for battery voltage should be more reasonable.  Having such a low bottom of 2.0v doesn't really make sense in the scenario that we're only boosting.  Maybe 3.2v would be a better minimum.  I guess having a high max value is okay as it lets the user boost a lot if they want - but it'll probably be initially confusing as people will wonder how their batteries could ever be at 5.0v/cell.

Reply:
Is it worth combining the two settings? i.e. if voltage is > 0 it's enabled, otherwise disabled?